### PR TITLE
title-dash only when connected

### PR DESCRIPTION
**Pull request by adrian-kong**

### CODE REVIEW

File: console_backend/src/status_bar.rs [hunk 1]
```diff
@@ -182,10 +182,12 @@ impl StatusBar {
...
-        status_bar_status.set_title(&format!(
-            "{} - Swift Console {}",
-            sb_update.port, sb_update.version
-        ));
+        let port = if sb_update.port.is_empty() {
+            ""
+        } else {
+            sb_update.port + " - "
+        };
+        status_bar_status.set_title(&format!("{port} - Swift Console {}", sb_update.version));
```
Comment (silverjam) [inline console_backend/src/status_bar.rs:190]:
```suggestion
        status_bar_status.set_title(&format!("{port}Swift Console {}", sb_update.version));
```

**Comment by swiftnav-travis** [comment]
Frontend and Release Workflow Started [here](https://github.com/swift-nav/toolbox-backend-ci/actions/runs/5051428132)

**Comment by silverjam** [comment]
Why is this going into your combinenotes branch?

**Comment by adrian-kong** [comment]
> Why is this going into your combinenotes branch?

was used to port forward the changes, these changes was only introduced in there.
maybe should split out branches
will change title to make more sense.

**Pull request merged**